airbyte-integrations/connectors/source-mongodb-v2/src/main/resources/spec.json

```diff
@@ -189,6 +189,19 @@
           "default": [],
           "order": 12,
           "group": "advanced"
+      },
+      "schema": {
+          "title": "Schema",
+          "description": "Hardcoded schema for each stream. If enabled, all streams must be annotated with a schema.",
```

Comment:
Why do all streams need a schema?  Why can't it be an override?

Reply:
Oops I forgot to delete this one!